### PR TITLE
Add support for MySQL > 5.6

### PR DESCRIPTION
**Pull request by hexfusion**
* Add ability to check mysql_server version with  db_version_mysql
* Handle deprecated mysql_install_db in mysql 5.7+ during initialization

@jonjensen PTAL

re: GH #32

### CODE REVIEW

File: lib/Camp/Master.pm [hunk 1]
```diff
@@ -2393,7 +2393,9 @@ EOF
 sub _initialize_camp_database_mysql {
     my $conf = shift;
     _render_database_config($conf)  unless ($conf->{_did_render_database_config});
-    my $cmd = "mysql_install_db --datadir=$conf->{db_data} --defaults-file=$conf->{db_conf}";
+    my $cmd  = (db_version_mysql() >= 5.7) ?
+      "mysqld  --defaults-file=$conf->{db_conf} --initialize-insecure --datadir=$conf->{db_data}" :
```
Comment (jonjensen) [inline lib/Camp/Master.pm]: @hexfusion Is the order of the command line arguments important? If not, I think they should be the same order between the two versions so it's obvious what is different between the two.

File: lib/Camp/Master.pm [hunk 2]
```diff
@@ -2843,6 +2851,21 @@ sub db_version_pg {
     return $major_minor;
 }
 
+sub db_version_mysql {
+    my %arg = @_;
+
+    # Some examples of $raw output:
+    # 5.7.15-9
+    # 10.1.20-MariaDB
```
Comment (jonjensen) [inline lib/Camp/Master.pm]: @hexfusion We don't really support MariaDB, do we? At any rate given its totally different version numbers it seems weird to treat it as the same thing here.

What versions of MySQL and/or MariaDB have you tested these changes with?

**Comment by hexfusion** [comment]
@jonjensen adding some tests will respond once complete.

ref: https://github.com/devcamps/camps/issues/34

**Comment by jonjensen** [comment]
I just went ahead and reordered the args. This is now merged.

File: lib/Camp/Master.pm (hunk 1 above)
Comment (hexfusion) [reply lib/Camp/Master.pm]: @jonjensen yeah order matters here sorry I missed this question.

**Pull request merged**